golang/cosmos/x/swingset/keeper/keeper.go

```diff
@@ -99,23 +100,23 @@ func (k Keeper) PushAction(ctx sdk.Context, action vm.Jsonable) error {
 
 	// JS uses IEEE 754 floats so avoid overflowing integers
 	if tail == MaxUint53 {
-		return errors.New("actionQueue overflow")
+		return errors.New(StoragePathActionQueue + " overflow")
 	}
 
 	// Set the vstorage corresponding to the queue entry for the current tail.
-	k.vstorageKeeper.SetStorage(ctx, fmt.Sprintf("actionQueue.%d", tail), string(bz))
+	k.vstorageKeeper.SetStorage(ctx, vstoragetypes.NewStorageEntry(fmt.Sprintf(StoragePathActionQueue+".%d", tail), string(bz)))
```

Comment:
Embedding a constant in a format template is bad form.
```suggestion
	path := StoragePathActionQueue + "." + strconv.FormatUint(tail, 10)
	k.vstorageKeeper.SetStorage(ctx, vstoragetypes.NewStorageEntry(path, string(bz)))
```